discord/topic.rb

```diff
@@ -1,7 +1,7 @@
 module Topic
   extend Discordrb::Commands::CommandContainer
 
-  command(:topic) do |event|
+  command(:topic, description: "Update's discord channel's topic to match that of the connected IRC channel.  Also displays channel modes.") do |event|
```

Comment:
```suggestion
  command(:topic, description: "Update's discord channel's topic to match that of the connected IRC channel. Also displays channel modes.") do |event|
```

Why's this space here